tests/test_srureader.py

```diff
@@ -9,7 +9,7 @@
 TESTDATA = json.load(open(Path(__file__).parent / 'TESTDATA', 'r'))
 
 
-class SRUReaderTestCase(unittest.TestCase):
+class TestSRUReader:
     @patch('edpop_explorer.srureader.sruthi')
```

Comment:
Since you are converting to pytest, is there a reason for keeping in this unit test function?

Reply:
I thought pytest is used for unit tests as well? Or do you suggest a different set-up of the tests?

Reply:
Oh, this was unclear. I meant that `patch` comes from the `unittest` library.